NSSNamespace/Initial_Bangazon_Site
# Button now disabled until you pick a payment type

Users now cannot Submit Order until the the payment type has been selected. 


Files changed:
Views/Order/Cart.cshtml 
wwwroot/js/site.js

TO TEST:
git fetch 
git checkout dy-orderDisable 

choose an active customer 
click on their name to go to their cart 
once on their cart, make sure the button is disabled until you pick a payment type.
